README.md

```diff
@@ -30,6 +30,13 @@ Looking for the old Hipster Shop frontend interface? Use the [manifests](https:/
 | ----------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------ |
 | [![Screenshot of store homepage](./docs/img/online-boutique-frontend-1.png)](./docs/img/online-boutique-frontend-1.png) | [![Screenshot of checkout screen](./docs/img/online-boutique-frontend-2.png)](./docs/img/online-boutique-frontend-2.png) |
 
+## Quickstart (GKE Autopilot)
```

Comment:
I agree with the deletion of this block, since step 3 is the only place users have to think about Standard vs Autopilot.
And as, William mentioned, you cover the details clearly in step 3.

Reply:
Good catches, accidental artifacts from previous edits.